package.json

```diff
@@ -31,38 +31,41 @@
   "dependencies": {
     "autoprefixer-stylus": "0.14.0",
     "cheerio": "0.22.0",
-    "chokidar": "1.7.0",
-    "handlebars": "4.0.10",
+    "chokidar": "^2.0.3",
```

Comment:
Nit: We now have some pinned dependencies and some not. Can we use the same versioning scheme for all dependencies?

Reply:
IMO, we should pin the dependencies.

Reply:
Sorry, updating …